app/assets/stylesheets/gmaps.css.scss

```diff
@@ -124,6 +124,21 @@
   }
 }
 
+strong.organisation-title {
+  color: #006937;
```

Comment:
for color values I suggest to use the following color [file](https://github.com/AgileVentures/LocalSupport/blob/develop/app/assets/stylesheets/utils/_colors.css.scss), it will be helpful in future to change color degrees or values from one location.